uiowa/uids
# Feature: Default table styles

Resolves #509
Resolves #510 

# How to test

https://uids.brand.uiowa.edu/branches/feature_default_tables/components/detail/tables.html

Check different table examples. The previous responsive behavior has been removed in favor of a horizontal scroll that is in effect below the `sm` breakpoint. Sticky headers (top or left) will no longer work. Compare the tables with the current tables to see if anything appears broken in appearance.

There is a hidden example of using a wrapping div for adding horizontal scrolling to larger tables beyond the `sm` breakpoint. To view it, set the responsive variant to not be hidden locally in `src/components/tables/tables.config.yml` and view the resulting page that is added to the documentation.
